package.json

```diff
@@ -15,10 +16,12 @@
     "node": ">=6.0.0"
   },
   "scripts": {
-    "build": "BABEL_ENV=production babel src -d dist",
-    "clean": "rm -rf dist",
+    "build:main": "BABEL_ENV=main babel src/index.js -o dist/index.umd.js",
```

Comment:
Hey @pgilad, NABD - this all looks good but just one small thing. I'd add the `build` script back in with: 

```
npm run clean && npm run build:main && npm run build:module && npm run build:es2015
```

And then on prepublishOnly just do:
```
npm run lint && npm run build
``` 

Makes the testing workflow easier, as I like to build all and then npm link this into my project to test.